src/main/java/com/hackathonorganizer/hackathonwriteservice/hackathon/exception/ResourceAlreadyExistsException.java

```diff
@@ -0,0 +1,8 @@
+package com.hackathonorganizer.hackathonwriteservice.hackathon.exception;
+
+public class ResourceAlreadyExistsException extends RuntimeException{
```

Comment:
imo prostsze podejście zrobić sobie bazowy exception typu ServiceException czy RestException który by dostawał HttpStatus w konstruktorze i na jego podstawie budować takie exceptiony, wtedy wystarczy jeden error handler ogarniający wszystko bez potrzeby mnożenia bytów tak jak to teraz jest z error handlerami per exception

Reply:
Z początku tak miałem to napisali mi żeby lepiej zrobić tak jest teraz, trzeba by tutaj też ustalić jakąś wspólną wersje